subsys/bluetooth/controller/ll_sw/ull_adv_types.h

```diff
@@ -60,5 +61,40 @@ struct ll_adv_sync_set {
 	uint8_t is_enabled:1;
 	uint8_t is_started:1;
 };
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO)
+struct ll_adv_iso {
+	struct evt_hdr        evt;
+	struct ull_hdr        ull;
+	struct lll_adv_iso    lll;
+
+	uint8_t  hci_handle;
+	uint16_t bis_handle; /* TODO: Support multiple BIS per BIG */
+
+	uint8_t  is_created:1;
+	uint8_t  encryption:1;
+	uint8_t  framing:1;
+	uint8_t  num_bis:5;
+
+	uint32_t sdu_interval:20;
+	uint16_t max_sdu:12;
+
+	uint16_t max_latency:12;
+
+	uint8_t  rtn:4;
+	uint8_t  phy:3;
+	uint8_t  packing:1;
+
+	uint8_t  bcode[16];
+
+	struct node_rx_hdr node_rx_complete;
+	struct {
+		struct node_rx_hdr node_rx_hdr_terminate;
+		uint8_t reason;
+	} node_rx_terminate;
```

Comment:
Please add alignment and comments here similar to `node_rx_lost` in `ll_sync_set`.